packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,6 +6,23 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## February
 
+### Weekly release 2022.02.1
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [API token ID added to System Log Event types](#api-token-id-added-to-system-log-event-types)                        | February 9, 2022         |
+| [Bug fixed in 2022.02.1](#bug-fixed-in-2022-02-1)                        | February 9, 2022         |
+
+#### API token ID added to System Log Event types
+
+API requests that include an API token and return a System Log event now include the API token in the event payload. The token identifier appears in the System Log API `transaction.details.requestApiTokenId` field and in the `Event > System > Transaction > Detail > RequestApiTokenId` node in the Admin Console System Log.
```

Comment:
In the heading you have Event, but here you have event. Should they all be event?

```suggestion
API requests that include an API token and return a System Log event now include the API token in the event payload. The token identifier appears in the System Log API `transaction.details.requestApiTokenId` field and in the `Event > System > Transaction > Detail > RequestApiTokenId` node in the Admin Console System Log.
```

Reply:
Thanks! Updated as per our Slack.